executor/common_linux.h

```diff
@@ -798,6 +798,509 @@ static long syz_extract_tcp_res(long a0, long a1, long a2)
 }
 #endif
 
+#if SYZ_EXECUTOR || __NR_syz_usb_connect
```

Comment:
Let's move all of usb syscalls to a separate file like we do for kvm.
common_linux.h is already unmanageably large, we probably need to move more code out of it, but for now let's at least not do even larger.
Note: if you add another header file you will need to add it to pkg/csource/gen.go as well.

Reply:
Done.